app/controllers/all/handover/handovers_controller.rb

```diff
@@ -41,6 +43,14 @@ def create
     end
   end
 
+  private def bail_missing_incoming_trust_ukprn
```

Comment:
Is 'bail' conventional naming in this project for this type of error - I can't find it used elsewhere

Reply:
Fab - I don't mean to nitpick, just trying to learn about the project :)